rcl/test/rcl/client_fixture.cpp

```diff
@@ -24,6 +24,33 @@
 #include "../memory_tools/memory_tools.hpp"
 #include "../scope_exit.hpp"
 #include "rcl/error_handling.h"
+#include "rcl/graph.h"
+
+bool
+wait_for_server_to_be_available(
+  rcl_node_t * node,
+  rcl_client_t * client,
+  size_t max_tries,
+  int64_t period_ms)
```

Comment:
The argument uses the same type as the already existing function. Therefore I would prefer to keep it similar.